docs/diagrams/ExportCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,101 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":TaAssistParser" as TaAssistParser LOGIC_COLOR
+participant "e:ExportCommand" as ExportCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
```

Comment:
This class is reused in the puml. I think it should be duplicated instead. 
Currently the lifeline of CommandResult doesn't look right in the generated image.

Reply:
Fixed in the next commit by creating 2 separate instances of ExportCommand instead.